core/src/main/java/io/seata/core/rpc/netty/AbstractNettyRemoting.java

```diff
@@ -289,9 +288,9 @@ protected void processMessage(ChannelHandlerContext ctx, RpcMessage rpcMessage)
                         if (allowDumpStack) {
                             String name = ManagementFactory.getRuntimeMXBean().getName();
                             String pid = name.split("@")[0];
-                            int idx = new Random().nextInt(100);
+                            long idx = System.currentTimeMillis();
                             try {
-                                Runtime.getRuntime().exec("jstack " + pid + " >d:/" + idx + ".log");
+                                Runtime.getRuntime().exec("jstack " + pid + " > " + idx + ".log");
```

Comment:
may be no permission, use current execute user home dir?

Reply:
and log.info the command?

Reply:
> may be no permission, use current execute user home dir?

I think this operation will  dump the file to the folder of the running path?

Reply:
recommended to add logging record commands. view the stack information path from the log information.

Reply:
recommended to add logging record commands. view the stack information path from the log information.

Reply:
done